packages/components-css/accordion/index.scss

```
@@ -0,0 +1,7 @@
+@import "../node_modules/@utrecht/components/accordion/src/";
+
+.utrecht-accordion__button {
+  @include utrecht-accordion__button;
+
+  --utrecht-button-border-radius: var(--utrecht-accordion-border-radius);
```

```suggestion
  --utrecht-button-border-radius: var(--utrecht-accordion-border-radius);
```

Ik verwacht dat de include niet nodig is, want die is al onderdeel van de file die je `@import`